frontend/viewer/src/project/browse/EntryView.svelte

```diff
@@ -51,6 +53,14 @@
 
   let readonly = $state(false);
   const entryPersistence = new EntryPersistence(() => entry, () => void entryResource.refetch());
+
+  const loadedEntryId = $derived(entry?.id);
+  let entryScrollViewportRef: HTMLElement | null = $state(null);
+  let editorRef: HTMLElement | null = $state(null);
+  watch([() => loadedEntryId, () => entryScrollViewportRef, () => editorRef], () =>{
```

Comment:
might I suggest `watch(() => [loadedEntryId, entryScrollViewportRef, editorRef], () => {...}))`, not sure why they have this option as it's so noisy.